updated Dashboard, Pivot EUD 20.1 (#107)
DevExpress/dotnet-eud: interface-elements-for-web/articles/pivot-table/data-presentation/filter-data-in-pivot-tables.md

# Filter Data in Pivot Tables
By default, the Pivot Table processes all records when calculating its values. Data filtering allows you to process only those records that meet your requirements.

> Topics in this section describe two ways of filtering data in Pivot Tables - using Filter Windows (provide simple filtering capabilities) and the Filter Editor (used for advanced filtering).

Reviewer: ```suggestion
This section explains how to use a Filter Window (for simple filtering) and a Filter Editor (for advanced filtering) to filter Pivot Table data.
```

The following sections and topics are available:
* [Filter Data by Field Values](filter-data/filter-data-by-field-values.md)
	
	This section contains topics that describe how to specify simple filter conditions via Filter Windows.
* [Build Complex Filter Criteria](filter-data/build-complex-filter-criteria.md)
	

> 	This topic describes the Filter Editor feature that allows you to build complex filter criteria. 

Reviewer: ```suggestion
	This topic describes the Filter Editor feature. This feature allows you to build complex filter criteria. 
```
